cmd/installer/cmd/image.go

```diff
@@ -148,6 +148,10 @@ func finalize(platform runtime.Platform, img string) (err error) {
 		if err = ova.CreateOVAFromRAW(name, img, outputArg); err != nil {
 			return err
 		}
+	case "metal":
```

Comment:
should we call it generic? ???

Reply:
it has to be a valid platform, and we don't have a `generic` one